short-read-mngs/idseq-dag/idseq_dag/steps/download_accessions.py

```diff
@@ -31,7 +24,12 @@ def run(self):
         # db_type = self.additional_attributes["db_type"]
         (_read_dict, accession_dict, _selected_genera) = m8.summarize_hits(hit_summary)
         with open_file_db_by_extension(loc_db, "QII", stringify=False) as loc_dict:
-            self.download_ref_sequences_from_file(accession_dict, loc_dict, db_s3_path, output_reference_fasta)
+            db_path = s3.fetch_reference(
```

Comment:
since you changed the db type to File won't this automatically get downloaded by miniwdl?

Reply:
Yeah iirc the `fetch_reference` code detects that and won't re-download. I think I can technically remove this but I just did a full revert for simplicity's sake.

Reply:
Yeah just confirmed this is the case.